src/main/java/com/actiontech/dble/manager/response/ShowSysParam.java

```diff
@@ -148,6 +148,9 @@ public static void execute(ManagerConnection c) {
         paramValues.add(sysConfig.getMaxCharsPerColumn() + "");
         paramValues.add(sysConfig.getMaxRowSizeToFile() + "");
         paramValues.add(sysConfig.isUseOuterHa() + "");
+        paramValues.add(sysConfig.isEnableFlowControl() + "");
```

Comment:
PARAM_NAMES
PARAM_DESCRIPTION
These two object also need to add param into

Reply:
Yes, I will add it.